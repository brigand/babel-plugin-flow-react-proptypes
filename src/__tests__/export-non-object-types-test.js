const babel = require('babel-core');
const content = `
export type Answer = "Yes" | "No";
`;

it('export-non-object-types', () => {
  const res = babel.transform(content, {
    babelrc: false,
    presets: ['es2015', 'stage-1', 'react'],
    plugins: ['syntax-flow', require('../')],
  }).code;
  expect(res).toMatchSnapshot();
});
