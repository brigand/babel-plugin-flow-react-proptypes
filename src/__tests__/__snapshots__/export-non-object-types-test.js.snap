// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`export-non-object-types 1`] = `
"\\"use strict\\";

var babelPluginFlowReactPropTypes_proptype_Answer = require(\\"prop-types\\").oneOf([\\"Yes\\", \\"No\\"]);

if (typeof exports !== \\"undefined\\") Object.defineProperty(exports, \\"babelPluginFlowReactPropTypes_proptype_Answer\\", {
  value: babelPluginFlowReactPropTypes_proptype_Answer,
  configurable: true,
  enumerable: true
});"
`;
